src/project/task-tree.test.js

```
@@ -0,0 +1,236 @@
+      const provider = new ProjectTasksTreeProvider(123, ['env1'], [], 'env1', false);
+      expect(provider.id).toBe(123);
+      expect(provider.envs).toEqual(['env1']);
+      expect(provider.tasks).toEqual([]);
+      expect(provider.selectedEnv).toBe('env1');
+      expect(provider.multiEnvProject).toBe(false);
+      expect(provider.multiEnvExplorer).toBe(false);
+    });
+  });
+
+  describe('getEnvTasks', () => {
+    it('returns tasks matching env and merges non-multienv tasks', () => {
```

ESLint

[error] 'it' is not defined.

(no-undef)